x-pack/plugins/ml/public/data_frame/pages/job_management/components/job_list/action_delete.tsx

```diff
@@ -103,7 +103,7 @@ export const DeleteAction: SFC<DeleteActionProps> = ({ deleteJob, item }) => {
           >
             <p>
               {i18n.translate('xpack.ml.dataframe.jobsList.deleteModalBody', {
-                defaultMessage: 'Are you sure you want to delete this job?',
+                defaultMessage: `Are you sure you want to delete this job? The job's target index and optional Kibana index pattern will not be delete.`,
```

Comment:
Typo - should be `deleted` rather than `delete` at the end.

Reply:
Fixed in 27548ed.